src/test/javascript/spec/component/iris/settings/iris-settings-update-component.spec.ts

```
@@ -48,14 +47,7 @@ describe('IrisSettingsUpdateComponent', () => {
         };
 
         TestBed.configureTestingModule({
-            imports: [
-                ArtemisTestModule,
-                NgbTooltipMockDirective,
-                MockJhiTranslateDirective,
-                IrisCourseSettingsUpdateComponent,
-                IrisSettingsUpdateComponent,
-                IrisCommonSubSettingsUpdateComponent,
-            ],
+            imports: [ArtemisTestModule, , MockJhiTranslateDirective, IrisCourseSettingsUpdateComponent, IrisSettingsUpdateComponent, IrisCommonSubSettingsUpdateComponent],
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)